core/tee/tee_svc_cryp.c

```diff
@@ -3839,15 +3999,22 @@ TEE_Result syscall_authenc_init(unsigned long state, const void *nonce,
 		return TEE_ERROR_BAD_PARAMETERS;
 
 	key = o->attr;
+
+	res = bb_memdup_user(nonce, nonce_len, &nonce_bbuf);
```

Comment:
can remove call to `vm_check_access_rights()` at line 3984 as processed here.

Reply:
updated